packages/suite/src/views/recovery/steps/EnterOnDeviceStep.tsx

```diff
@@ -1,9 +1,28 @@
-import { Paragraph } from '@trezor/components';
+import { DEFAULT_FLAGSHIP_MODEL } from '@suite-common/suite-constants';
+import { Banner, IconName, Paragraph } from '@trezor/components';
+import { spacings } from '@trezor/theme';
 
 import { Translation } from 'src/components/suite';
+import { useDevice } from 'src/hooks/suite';
 
-export const EnterOnDeviceStep = () => (
-    <Paragraph>
-        <Translation id="TR_ENTER_SEED_WORDS_ON_DEVICE" />
-    </Paragraph>
-);
+export const EnterOnDeviceStep = () => {
+    const { device } = useDevice();
+    const deviceModelInternal = device?.features?.internal_model || DEFAULT_FLAGSHIP_MODEL;
+
+    return (
+        <Banner
+            variant="info"
+            icon={
+                deviceModelInternal === 'UNKNOWN'
+                    ? undefined
+                    : (`trezor${deviceModelInternal}` as IconName) // TODO: update to non-deprecated icons
```

Comment:
Unnecessary `as`, it passes type check
```suggestion
                    : (`trezor${deviceModelInternal}`) // TODO: update to non-deprecated icons
```